test/core-test.cc

```diff
@@ -213,12 +213,12 @@ struct custom_context {
 
   template <typename T> struct formatter_type {
     template <typename ParseContext>
-    auto parse(ParseContext& ctx) -> decltype(ctx.begin()) {
-      return ctx.begin();
+    auto parse(ParseContext& parse_ctx) -> decltype(parse_ctx.begin()) {
```

Comment:
Let's keep argument names as is but rename the data member to `parse_ctx` since it has wider scope.

Reply:
Done - I'm glad you looked at that one, I had a feeling that the situation might be something like this but I didn't want to be too aggressive so I tried my best to keep the scope very local.